### PR TITLE
implement normalizeForVisualization for BoundCRS

### PR DESCRIPTION
**Pull request by jjimenezshaw**
Just adding this missing implementation 

- [x] Tests added
 - [x] Added clear title that can be used to generate release notes

### CODE REVIEW

**Comment by rouault** [comment]
Did you check that the transformation associated with the boundCRS still works OK after normalizeForVisualization() ? I'd think so, but worth checking if not done

**Comment by jjimenezshaw** [comment]
> Did you check that the transformation associated with the boundCRS still works OK after normalizeForVisualization() ? I'd think so, but worth checking if not done

I added a check of `exportToPROJString`. Is that what you mean? the output of `exportToPROJString` is the same before and after `normalizeForVisualization`.

**Comment by rouault** [comment]
> I added a check of `exportToPROJString`. Is that what you mean? the output of `exportToPROJString` is the same before and after `normalizeForVisualization`.

I was thinking more to the transformation pipeline returned by createOperation() from the bound CRS to EPSG:4326.  exportToPROJString() on the boundCRS itself isn't very significant as PROJ CRS strings don't capture axis order.

**Comment by jjimenezshaw** [comment]
>  I was thinking more to the transformation pipeline returned by createOperation()

That was my other option. I'll do it later.

**Comment by jjimenezshaw** [comment]
MacOS CI is failing on something that I guess is not related with my changes.

**Comment by rouault** [comment]
> MacOS CI is failing on something that I guess is not related with my changes.

agreed. PR merged

File: src/iso19111/crs.cpp [hunk 1]
```diff
@@ -1169,6 +1169,12 @@ CRSNNPtr CRS::normalizeForVisualization() const {
         }
     }
 
+    if (const BoundCRS *boundCRS = dynamic_cast<const BoundCRS *>(this)) {
+        auto baseNormCRS = boundCRS->baseCRS()->normalizeForVisualization();
+        return BoundCRS::create(baseNormCRS, boundCRS->hubCRS(),
+                                boundCRS->transformation());
+    }
```
Comment (hernando) [inline src/iso19111/crs.cpp:1176]: Thanks for this fix. You forgot to simplify the code that calls this function assuming it doesn't handle BoundCRS.

Comment (hernando) [reply src/iso19111/crs.cpp:1176]: Ignore this comment, I got confused by other code base.

**Pull request merged**